src/test/fixtures/apps/archive/codeversion/contents/testbook1-shortid:testpage12-shortid.json

```diff
@@ -2,6 +2,6 @@
left\">Bond Energy</th> </tr> </thead> <tbody> <tr valign=\"top\"> <td data-align=\"left\">H–H</td> <td data-align=\"left\">436</td> <td data-align=\"left\" rowspan=\"23\" /> <td data-align=\"left\">C–S</td> <td data-align=\"left\">260</td> <td data-align=\"left\" rowspan=\"23\" /> <td data-align=\"left\">F–Cl</td> <td data-align=\"left\">255</td> </tr> <tr valign=\"top\"> <td data-align=\"left\">H–C</td> <td data-align=\"left\">415</td> <td data-align=\"left\">C–Cl</td> <td data-align=\"left\">330</td> <td data-align=\"left\">F–Br</td> <td data-align=\"left\">235</td> </tr> <tr valign=\"top\"> <td data-align=\"left\">H–N</td> <td data-align=\"left\">390</td> <td data-align=\"left\">C–Br</td> <td data-align=\"left\">275</td> <td data-align=\"left\">Si–Si</td> <td data-align=\"left\">230</td> </tr> <tr valign=\"top\"> <td data-align=\"left\">H–O</td> <td data-align=\"left\">464</td> <td data-align=\"left\">C–I</td> <td data-align=\"left\">240</td> <td data-align=\"left\">Si–P</td> <td data-align=\"left\">215</td> </tr> <tr valign=\"top\"> <td data-align=\"left\">H–F</td> <td data-align=\"left\">569</td> <td data-align=\"left\">N–N</td> <td data-align=\"left\">160</td> <td data-align=\"left\">Si–S</td> <td data-align=\"left\">225</td> </tr> </tbody> </table> <div class=\"os-caption-container\"><span class=\"os-title-label\">Table </span><span class=\"os-number\">7.2</span><span class=\"os-divider\"> </span><span class=\"os-divider\"> </span></div> </div> <div class=\"os-eoc os-glossary-container\" data-type=\"composite-page\" data-uuid-key=\"glossary\"> <dl id=\"fs-idm397777376\"> <dt id=\"18469\">business ethics</dt> <dd id=\"fs-idm406937056\">the conduct by which companies and their agents abide by the law and respect the rights of their stakeholders, particularly their customers, clients, employees, and the surrounding community and environment</dd> </dl> <dl id=\"fs-idm407518992\"> <dt id=\"34873\">compliance</dt> <dd id=\"fs-idm405304016\">the extent to which a company conducts its business operations in accordance with applicable regulation and statutes</dd> </dl> <dl id=\"fs-idm403168464\"> <dt id=\"67088\">corporate culture</dt> <dd id=\"fs-idm401618864\">the shared beliefs, values, and behaviors that create the organizational context within which employees and managers interact <a data-type=\"cite\" href=\"\"><sup class=\"os-end-note-number\">20</sup></a> </dd> </dl> </div> <div id=\"table-of-links\"> <h3 class=\"os-title\" data-type=\"title\">Table of Links</h3> <ul> <li> <a href=\"#main-content\">Main Content</a> </li> <li> <a href=\"#fs-id1166227577868\">A note</a> </li> <li> <a href=\"#term104\">A term</a> </li> <li> <a href=\"#fs-id1166227682186\">A blockquote</a> </li> <li> <a href=\"#fs-id1166227571565\">A paragraph</a> </li> <li> <a href=\"#CNX_Chem_06_00_CrabNeb\">A figure</a> </li> <li> <a href=\"#fs-idp13638832\">A table</a> </li> <li> <a href=\"#fs-idm397777376\">A glossary term</a> </li> </ul> </div> </div> </body>\n </html>",
```

Comment:
the `src` in the content is a relative url, we transform it to rebase it, the previous fixture was correct

Reply:
The reason I don't get code coverage is that no tests have images with absolute paths beginning with /apps/archive. Do I need to create a new fixture or add a new image to an existing fixture? Or something else?